src/externs.ts

```diff
@@ -681,17 +713,13 @@ export function generateExterns(
       case ts.SyntaxKind.ImportEqualsDeclaration:
         const importEquals = node as ts.ImportEqualsDeclaration;
         const localName = getIdentifierText(importEquals.name);
-        if (localName === 'ng') {
-          emit(`\n/* Skipping problematic import ng = ...; */\n`);
-          break;
-        }
         if (importEquals.moduleReference.kind === ts.SyntaxKind.ExternalModuleReference) {
           addImportAliases(importEquals);
           break;
         }
-        const qn = getEntityNameText(importEquals.moduleReference);
+        const qn = qualifiedNameToMangledIdentifier(importEquals.moduleReference);
         // @const so that Closure Compiler understands this is an alias.
-        if (namespace.length === 0) emit('/** @const */\n');
```

Comment:
why drop the if check?

Reply:
I don't actually know why we had the check there, I think it was inherited over many generations of this file. It's simply not right, we should mark all generated namespaces as `@const` so Closure can infer types better, and they can be used in `typeof` etc.